ci/generate_combinations.py

```diff
@@ -42,7 +42,7 @@ def iter_arches() -> Generator[dict, None, None]:
     logger.info("Fetching arches")
     archive_ids = list(iter_archive_ids(category="qt"))
     for archive_id in tqdm(archive_ids):
-        for version in ("latest", "5.15.2", "5.13.2", "5.9.9"):
+        for version in ("latest", "6.5", "6.2", "5.15.2", "5.13.2", "5.9.9"):
```

Comment:
Oops; this doesn't actually work because 6.5 and 6.2 are not valid versions. I was counting on CI to fail if the script doesn't work; I didn't realize it wasn't going to run.